src/database/statement.ts

```diff
@@ -8,4 +8,16 @@ export default class Statement {
       linesBetweenQueries: 2, // Defaults to 1
     });
   }
+
+  static delimName(name: string) {
+    if (name.startsWith(`"`) && name.endsWith(`"`)) return name;
+    if (name.includes(` `)) return `"${name}"`;
+    if (name.length <= 10) return name.toUpperCase();
+    else return `"${name}"`;
+  }
```

Comment:
With a method like this, the calling context is important.
Did the value come from an SQL catalog, or did the user type in the value?
Values from the catalog are already normalized, but user input could be anything.

For example, input value `create_sql_sample`
See test in `testing/database.ts` at line 71

If that value came from a catalog, then the case is very likely significant, and thus must be delimited in order to preserve it.
But if the value was typed in by the user, since they did not delimit the value, it should be folded to uppercase, because as it is it is a valid ordinary identifier which can be up to 128 characters long.

Reply:
> Did the value come from an SQL catalog, or did the user type in the value?

Yeah, I realized half-way through this impl. that I might need to do something like this. I am going to add an additional parameter to handle these cases.